test/controllers/relation_controller_test.rb

```diff
@@ -337,6 +337,7 @@ def test_create
   def test_update_relation_tags
     basic_authorization "test@example.com", "test"
     rel_id = current_relations(:multi_tag_relation).id
+    create_list(:relation_tag, 4, :relation => current_relations(:multi_tag_relation))
```

Comment:
I don't know what the original bug was that's hinted at in the comment. Since the test involves updating an existing tag I though it best to include more than one (the fixtures had four tags, so I stuck with that). As you say, it doesn't seem to matter that the db is in an initially inconsistent state, since after the update, it's the new version that is checked, not the old version that has no tags.

I can propagate the tags if you think it's worthwhile though.